docs/java-activities.md

```diff
@@ -0,0 +1,207 @@
+---
+id: java-activities
+title: Activities in Java
+sidebar_label: Activities
+---
+
+## Overview
+
+In Java, a Temporal Activity is simply an object method that contains any business-level logic.
```

Comment:
This is maybe not as informative as it could be? Workflows can and should probably have the majority of your business logic, while activities exist to do things workflows can't - like block, perform heavy computation, side effects, be non-deterministic, etc. Probably worth alluding to that distinction here as well.

Reply:
I am actually going to scrap the overview altogether.
The idea of "what" an Activity is should be in the concepts section. Otherwise, we are repeating ourselves over and over.